pkg/services/dashboardimport/api/api.go

```diff
@@ -43,9 +46,34 @@ func (api *ImportDashboardAPI) RegisterAPIEndpoints(routeRegister routing.RouteR
 			authorize(accesscontrol.EvalPermission(dashboards.ActionDashboardsCreate)),
 			routing.Wrap(api.ImportDashboard),
 		)
+		if api.features.IsEnabledGlobally(featuremgmt.FlagDashboardLibrary) {
+			route.Post(
+				"/interpolate",
+				authorize(accesscontrol.EvalPermission(dashboards.ActionDashboardsCreate)),
```

Comment:
🤔 Considering interpolate don't save a dashboards, we can use the read read only, viewers could also preview the templates and try the feature. 

p.s. I tested locally the current behaviour, without the template dashboards and  viewers can see the empty dashboard page, create panels, etc, but they can't save them

Reply:
I just checked that a viewer cannot see the dashboard empty page 🧐

Reply:
I see, I have a viewer user with access to one folder, that's the reason they can see the page. Do you see any risks on allowing viewers to try out templates?

https://github.com/user-attachments/assets/07fc5487-b91f-4999-a065-86dc27acce1e